### ISSUE TITLE
Show user avatars in group chats

### ISSUE THREAD

**Issue by mxmerz** [signalapp/Signal-iOS#321]
In a group chat, I find it too hard to quickly identify who wrote what. The bubbles are all grey, the names are all grey and there are no avatars shown. Especially if two of the chat partners have similar names, this gets difficult.

Proposal: Show the avatars of users besides their chat bubbles or color the bubbles or the names per user. Compare, for example, a group chat in Signal to a group chat in Threema or WhatsApp – I personally like the Threema approach with avatars better, but the WhatsApp solution also helps the user.

**Comment by mxmerz** [comment]
Is this feature still on the roadmap?

**Comment by michaelkirk** [comment]
I believe this was accidentally closed.

**Comment by pkosiec** [comment]
@michaelkirk @michaelkirk-signal @charlesmchen @charlesmchen-signal Hi, any progress on this? I've recently started a group conversation on Signal and this is definitely a must-have!

**Comment by scrapbird** [comment]
How has this issue been open for 3 years now? It’s extremely hard to follow group chats without one of these suggested solutions. The group chat UI on iOS is quite frankly piss poor in its current state.

**Comment by michaelkirk-signal** [comment]
We've made some changes to address this in the 2.28 release.
